e2e/utils.go

```diff
@@ -1266,3 +1274,38 @@ func validateSubvolumegroup(f *framework.Framework, subvolgrp string) error {
 	}
 	return nil
 }
+
+// upgradeCSI deploys a desired ceph-csi release version.
+func upgradeCSI(version string) error {
+	tempDir := "/tmp/csi"
```

Comment:
as an enhancement use `ioutil.TempDir` and save it some variable and we need to remove this directory after E2E. in aftersuite function

Reply:
Since we are cloning into /tmp, it gets cleaned automatically. Do we need to clean it manually?

Reply:
If we run multiple times with different versions on the same machine it's not the case, better to do cleanup in aftereach please